node/tests/src/main.rs

```diff
@@ -83,6 +83,15 @@ pub async fn sanity_test() {
     }
 }
 
+/// Sanity test for the RPC server.
+/// We use unwraps here because this function is intended to be used like a test.
+pub async fn delay_test() {
```

Comment:
This is not a delay test per se, but rather applying delay behavior on a running pod.

Reply:
Right, it was initially a rough draft just to observe how the delay functions. It should now operate as a proper test since it verifies a couple of conditions after the delay.